### DISCUSSION TITLE
How to create metallic reflection on surfaces?

### DISCUSSION THREAD

**Discussion by czettnersandor** [bevyengine/bevy#11255] [Q&A]
Hi, I'm currently enjoying Bevy for my side project and I have a question I can't solve the past few days. I want the coins to imitate a polished gold surface. I added EnvironmentMapLight, but it's making shadows barely visible, and it's giving me too much light.

If I remove the EnvironmentMapLight, then the metallic surfaces are just flat color. What can I do to make this work? I understand the metal needs to reflect something, but how can I do it without that something affecting the whole world?

I think I'll need a custom shader for this, but I'm not sure where to start.

See attached with and without EnvironmentMapLight to see what I mean. I added the balls to debug this issue easier:
![image](https://github.com/bevyengine/bevy/assets/263174/ffe1971f-3361-4220-968f-10b5c74fc119)
![image](https://github.com/bevyengine/bevy/assets/263174/23516339-9754-4ae1-92f0-830498e7f7c2)

**Comment by DavidHospital** [comment]
If you're using a `StandardMaterial` for your coins, try playing around with some of the [settings](https://docs.rs/bevy/latest/bevy/prelude/struct.StandardMaterial.html) (`metallic`, `reflectance`, etc.). The existing PBR shader for `StandardMaterial` should handle the complex lighting automatically.

**Comment by may-bell** [comment]
i'm running into this exact issue: controlling the influence `EnvironmentMapLight` has on the ambient light in a scene is something i'm struggling to find a solution i'm happy with. it's WAY too bright on the shadows, and way too dim on the metallicity.

the PBR shader is "just guessing" and creating flat, uninteresting images.

**Reply by may-bell** [reply]
be sure to remove any `AmbientLight` components when working with PBR stuff, it adds light to the shadows which can throw things off further.

**Reply by JMS55** [reply]
Remove ambient light, and then adjust `EnvironmentMapLight::intensity` (along with the strength of your direct lighting).

I would load things into blender, render using cycles (pathtracer), try getting it to look good, and then try matching that in Bevy.
